esx_service/utils/auth_api.py

```diff
@@ -107,19 +151,19 @@ def generate_tuple_from_vm_list(vm_list):
     if not vm_list:
         return None, [], []
     vms = []
-    error_info = ""
+    error_msg = ""
     not_found_vms = []
     for vm_name in vm_list:
         vm_uuid = vmdk_utils.get_vm_uuid_by_name(vm_name)
         if not vm_uuid:
             err =  "Cannot find vm_uuid for vm {0} ".format(vm_name)
             if err:
-                error_info = error_info + err
+                error_msg = error_msg + err
                 not_found_vms.append(vm_name) 
         vms.append((vm_uuid, vm_name))
     
-    if error_info:
-        return error_info, vms, not_found_vms
+    if error_msg:
+        return error_msg, vms, not_found_vms
```

Comment:
Function "generate_tuple_from_vm_list" is not a API, it is only used by other API function in the same file.  Only the API function will return error_info. error_info is a struct defined in error_code.py, which includes both field "code" and "msg".